Private/Invoke-Command.ps1

```diff
@@ -0,0 +1,19 @@
+function Invoke-Command {
+    [CmdletBinding()]
+    param (
+        [Parameter(mandatory=$true)]
+        [string]
+        $Command
+    )
+
+    try {
+        $result = [CommandResult]::new()
+        $output = Invoke-Expression "& $Command" 2> $null
+        $result.ExitCode = $lastexitcode
+        $result.Success = $?
```

Comment:
I think for `$?` to work correctly, it should be placed directly below `Invoke-Expression`.
```suggestion
        $result.Success = $?
        $result.ExitCode = $lastexitcode
```

Reply:
The code suggestion should start at line 12. Swap line 12 with line 13.

Reply:
I have made it so! Well spotted :)